tests/test_operator.py

```diff
@@ -498,7 +498,7 @@ def test_default_functions(self):
         grid = Grid(shape=(5, 6, 7))
         f = TimeFunction(name='f', grid=grid)
         g = Function(name='g', grid=grid)
-        op = Operator(Eq(g, g + f), dle=('advanced', {'openmp': False}))
+        op = Operator(Eq(f.forward, g + f), dle=('advanced', {'openmp': False}))
```

Comment:
why?

Reply:
``f`` would have worked too. 

But not ``g``. With ``g`` on the LHS, this is a fully parallel loop nest -- you may go in parallel even over time -- and heuristically the compiler doesn't do blocking if there is not at least one SEQUENTIAL Iteration

the test however expects `x0_blk_size`, ie it expects blocking, so...